### ISSUE TITLE
"You cannot have any more summons" bug

### ISSUE THREAD

**Issue by jazzadellic** [Growlith1223/ArsMagica2#359]
Playing the Crash of the Titans modpack, and one of the AM2 summon spells I have which summons an undead type mob was working great for a while, and then all of a sudden it started saying "You cannot have any more summons". I would always make sure to kill any summons I had when done using them. Saving & quitting and completely restarting the game (+rebooting computer) doesn't help. I still get that message and the spell is now completely unusable. I didn't create the spell as it was a reward in a lootbag, so I don't know precisely the shapes used.

**Comment by TheIcyOne** [comment]
Yeah, summons are bugged in the current version. It's being worked on, but the code for the summons is a bit of a mess for current versions.

**Comment by EdwinMindcraft** [comment]
A workaround would be to manually change your summon count for now. You can see it in your player.dat file, under ForgeCaps.
